packages/zoom-in/src/components/zoom.js

```diff
@@ -144,6 +142,15 @@ const Zoom = props => {
     if (!zoomedRef.current) return
 
     setZoom(false)
+    overlayRef.current.cursor = ''
```

Comment:
Question 🙋‍♂️ :
Is `overlayRef.current.cursor` === `overlayRef.current.setAttribute` ?

Reply:
No, this is my fault.

`overlayRef.current.setAttribute` is correct statement.